astrocut/cube_cut.py

```diff
@@ -710,28 +710,30 @@ def _build_tpf(self, product, cube_fits, img_cube, uncert_cube, cutout_wcs_dict,
         dims = str(img_cube[0].shape[::-1])
         empty_arr = np.zeros(img_cube.shape)
         # Adding the Time relates columns
-        start = 'TSTART' if product == 'SPOC' else 'STARTTJD'
-        stop = 'TSTOP' if product == 'SPOC' else 'ENDTJD'
+        start = 'TSTART' if self.product == 'SPOC' else 'STARTTJD'
+        stop = 'TSTOP' if self.product == 'SPOC' else 'ENDTJD'
         cols.append(fits.Column(name='TIME', format='D', unit='BJD - 2457000, days', disp='D14.7',
                                 array=(cube_fits[2].columns[start].array + cube_fits[2].columns[stop].array)/2))
 
-        if product == 'SPOC':
+        if self.product == 'SPOC':
             cols.append(fits.Column(name='TIMECORR', format='E', unit='d', disp='E14.7',
                                     array=cube_fits[2].columns['BARYCORR'].array))
 
         # Adding CADENCENO as zeros for SPOC b/c we don't have this info
-        cadence_array = empty_arr[:, 0, 0] if product == 'SPOC' else cube_fits[2].columns['CADENCE'].array
+        cadence_array = np.zeros(img_cube.shape)[:, 0, 0] if self.product == 'SPOC'\
```

Comment:
There are other calls to `error_arr` besides flux_err_array, so I decided to just remove the assignment of error_array to cadence_array entirely. @falkben